### PR TITLE
docs: update example for Deno.Process

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by caspervonb — APPROVED** [review]
Geez, interface showing the pain points when put like that.

LGTM.

**Review by ry — APPROVED** [review]
LGTM

File: cli/dts/lib.deno.ns.d.ts [hunk 1]
```diff
@@ -1910,7 +1910,29 @@ declare namespace Deno {
       : (Reader & Closer) | null;
     readonly stderr: T["stderr"] extends "piped" ? Reader & Closer
       : (Reader & Closer) | null;
-    /** Resolves to the current status of the process. */
+    /** Wait for the process to exit and return its exit status.
+     *
+     * Calling this function multiple times will return the same status.
+     *
+     * Stdin handle to the process will be closed before waiting to avoid
+     * a deadlock.
```
Comment (ry) [inline cli/dts/lib.deno.ns.d.ts:1918]: Missing verb here

Comment (bartlomieju) [reply cli/dts/lib.deno.ns.d.ts:1918]: What do you mean?

**Pull request merged**